content/java/concepts/calendar/terms/add/add.md

```diff
@@ -0,0 +1,67 @@
+---
+Title: '.add()'
+Description: 'Adds or subtracts a specified amount of time to/from the given Calendar field.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Calendar'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.add()`** method of the `Calendar` class is used to manipulate and modify a `Calendar` instance by adding or subtracting a specified amount of time to/from a specfic Calendar field.
+
+## Syntax
+
+```pseudo
+myCalendar.add(field, amount)
+```
+
+- `field`: An integer constant representing the field to which the amount of time should be added or subtracted. This can be one of the following constants:
+    - `Calendar.YEAR`: Adds or subtracts the specified number of years.
+    - `Calendar.MONTH`: Adds or subtracts the specified number of months.
+    - `Calendar.DAY_OF_MONTH`: Adds or subtracts the specified number of days.
+    - `Calendar.HOUR_OF_DAY`: Adds or subtracts the specified number of hours.
+    - `Calendar.MINUTE`: Adds or subtracts the specified number of minutes.
+    - `Calendar.SECOND`: Adds or subtracts the specified number of seconds.
+    - `Calendar.MILLISECOND`: Adds or subtracts the specified number of milliseconds.
+
+- `amount`: An integer value representing the amount of time to add or subtract from the specified field. A positive value adds time, and a negative value subtracts time.
+
+## Example
+
+In the example below, we demonstrate how to use the `.add()` method to manipulate a `Calendar` instance:
+
+```java
+import java.util.Calendar;
+
+public class Main {
```

Comment:
The result of the operations might be clearer if you include a print call after each change